src/app/screens/LogIn/index.js

```diff
@@ -1,31 +1,38 @@
 import React from 'react';
 
+import { validateUser } from '../../../service/service';
+
 import LogInLayout from './layout';
 import './styles.css';
 
 class LogIn extends React.Component {
   validateLogIn = () => {
-    document.getElementById('password-validation').style.display = 'none';
-    document.getElementById('user-validation').style.display = 'none';
+    this.showErrorMessage('password-validation', 'none');
+    this.showErrorMessage('user-validation', 'none');
+    this.showErrorMessage('api-validation', 'none');
     const user = document.getElementById('user').value;
     const password = document.getElementById('password').value;
     let validationFlag = true;
     if (!this.validatePassword(password)) {
-      document.getElementById('password-validation').style.display = 'block';
+      this.showErrorMessage('password-validation', 'block');
       validationFlag = false;
     }
     if (!this.validateEmail(user)) {
-      document.getElementById('user-validation').style.display = 'block';
+      this.showErrorMessage('user-validation', 'block');
```

Comment:
move all constants to `Login/constants.js`

user-validation, password-validation, api-validation